src/main/webapp/gast/person.jsp

```diff
@@ -43,18 +43,24 @@
   <table id="personen-table" class="content-table">
 	<tbody>
 		<tr>
+                    <tr>
```

Comment:
There seems to be a duplicate `<tr>` start tag in here now

Reply:
done